fastly/kafka.go

```diff
@@ -29,6 +29,11 @@ type Kafka struct {
 	CreatedAt         *time.Time `mapstructure:"created_at"`
 	UpdatedAt         *time.Time `mapstructure:"updated_at"`
 	DeletedAt         *time.Time `mapstructure:"deleted_at"`
+	ParseLogKeyvals   bool       `mapstructure:"parse_log_keyvals"`
+	RequestMaxBytes   uint       `mapstructure:"request_max_bytes"`
+	AuthMethod        string     `mapstructure:"auth_method"`
+	User              string     `mapstructure:"user"`
+	Password          string     `mapstructure:"password"`
```

Comment:
Minor nit, the project convention is to keep the metadata fields `CreatedAt` etc as the last in the struct.

Reply:
Thanks, I fixed that up.